salt/journal/init.sls

```diff
@@ -169,9 +160,35 @@ journal-nginx-vhost:
             - service: nginx-server-service
             - service: php-fpm
 
+journal-nginx-vhost-dummy:
+    file.managed:
+        - name: /etc/nginx/sites-enabled/journal-dummy.conf
+        - source: salt://journal/config/etc-nginx-sites-enabled-journal-dummy.conf
+        - template: jinja
+        - require:
+            - nginx-config
+            - nginx-error-pages
+        - listen_in:
+            - service: nginx-server-service
+            - service: php-fpm
+
+running-gulp:
+    cmd.script:
+        - name: retrying-gulp
+        - source: salt://journal/scripts/retrying-gulp-without-redis.sh
+        - cwd: /srv/journal
+        - user: {{ pillar.elife.deploy_user.username }}
+        - require:
+            - journal-npm-install
+            - journal-node-modules-manual-install
+            - composer-install
+            - journal-nginx-vhost-dummy
+
 maintenance-mode-end:
     cmd.run:
-        - name: /etc/init.d/nginx start
+        - name: |
+            ln -s /etc/nginx/sites-available/journal.conf /etc/nginx/sites-enabled/journal.conf
+            /etc/init.d/nginx restart
```

Comment:
Thought that was the behaviour of `listen_in`. But anyway, guess this is a guarantee.